tests/testthat/test-sendData.R

```diff
@@ -77,11 +77,12 @@ test_that("sendData can operate on CredentialsList", {
   res <- dbtools::sendQuery(cred, "SELECT * FROM `mtcars`;", simplify = FALSE)
 
   # there should be two identical instances of mtcars in the result set
-  testthat::expect_identical(res[[2]][[1]], res[[1]][[1]])
-  testthat::expect_identical(res[[1]][[1]], tibble::as_data_frame(mtcars))
+  testthat::expect_identical(res[[2]], res[[1]])
+  testthat::expect_identical(res[[1]], tibble::as_data_frame(mtcars))
 
```

Comment:
@JonathanBob in der jetzigen Version versuche ich das Ergebnis zu vereinfachen, um keine geschachtelten Listen zurückzugeben, sondern nach Möglichkeit eine flache Struktur. Selbst beim `simplify = FALSE`; der Wert verhindert dadurch nur noch einen `bind_rows`. Zu sehen an den geänderten Tests. Ich halte dieses Verhalten für hilfreich, da wir sonst sehr oft den gleichen Code schreiben um die Rückgabewerte zu vereinfachen: Wer arbeitet schon gerne mit geschachtelten Listen...